release_notes/ocp-4-12-release-notes.adoc

```diff
@@ -164,6 +164,19 @@ The `console-operator` now scans all nodes and builds a set of all architecture
 
 For more information about multi-architechture compute machines, see xref:../post_installation_configuration/multi-architecture-configuration.adoc[Configuring a multi-architecture compute machine on an OpenShift cluster].
 
+[id="ocp-4-12-dynamic-plug-in-updates"]
+===== Dynamic plug-in generally available
+
+This feature was previously introduced as a Technology Preview in {product-title} 4.10 and is now generally available in {product-title} 4.12. With the dynamic plug-in you can build high quality and unique user experiences natively in the web console. You can:
```

Comment:
Do we need a comma here? _With the dynamic plug-in,_
I probably would put one; but I tend to over-comma things. Up to you